src/braket/default_simulator/simulator.py

```diff
@@ -345,7 +346,7 @@ def run_openqasm(
                 are requested when shots>0.
         """
         is_file = openqasm_ir.source.endswith(".qasm")
-        interpreter = Interpreter()
+        interpreter = Interpreter(context=ProgramContext())
```

Comment:
This shouldn't be necessary, right? I'd like `Interpreter()` to work as expected with default arg values